jaxrs-clients/src/test/java/feign/TextDelegateDecoderTest.java

```diff
@@ -78,6 +79,13 @@ public void testUsesStringDecoderWithTextPlain() throws Exception {
     }
 
     @Test
+    public void testCannotReturnStringWithMediaTypeJson() throws Exception {
+        assertThatThrownBy(() -> service.getJsonString("foo"))
```

Comment:
(Feign throwing the exception means that the content `foo` came back over the wire)